src/main/java/pl/piekoszek/gorskimatches/token/AccountInfo.java

```diff
@@ -0,0 +1,17 @@
+package pl.piekoszek.gorskimatches.token;
+
+import org.springframework.stereotype.Component;
+
+@Component
```

Comment:
This class doesn't do anything, it's "data transfer object", should not be `@Component`

Reply:
ok